internal/controller/management_controller.go

```diff
@@ -49,20 +50,22 @@ import (
 
 // ManagementReconciler reconciles a Management object
 type ManagementReconciler struct {
```

Comment:
not sure why client is unexported and DynamicClient is exported?

Reply:
That is because dynamic client is [being set in main.go](https://github.com/k0rdent/kcm/blob/5c0ce3213dfc2c989e9ee4efeb266d0ee34fb21b/cmd/main.go#L244) so it needs to be exported. Although I guess I can create the dynamic client in the SetupWithManager function as well, that would allow it to be unexported. I'll do that

Reply:
I am pretty sure that even if you make all fields exported this won't have any impact as code is behind `internal` folder.

Reply:
that does not matter, i've `super nit`'ed it bc of personal preference. no impact at all, I'm okay with either exported or unexported

let me know when the PR is ready to be merged

Reply:
I mean either way is ok, just let's be consistent with clients (both exported or both unexported)